### DISCUSSION TITLE
Call to a member function get() on null

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#15649] [General]
![call a member](https://user-images.githubusercontent.com/75428743/150392858-15f6b2c4-d22f-41a5-8b76-d792ae47ba5f.PNG)
Hello, I have such a problem with loading the admin username.

Version : 6.1.0.332
Ngnix
mysql 5.7.27

**Comment by mariuszkrzaczkowski** [comment]
https://github.com/YetiForceCompany/YetiForceCRM/discussions/15377#discussion-3654060
